cmd/runc-wrapper/runc-wrapper.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+)
+
+const (
+	runc        string = "/usr/libexec/docker/docker-runc-current"
+	logLocation string = "/tmp/nspawn-runc-wrapper.log"
+)
+
+func main() {
+	var newArgs []string
+
+	fd, err := os.OpenFile(logLocation, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0660)
+	if err != nil {
+		log.Println("error opening logs, skipping:", err)
+	}
+	log.SetOutput(fd)
+	defer fd.Close()
+
+	log.Printf("old args: %+v", os.Args[1:])
+
+	for _, a := range os.Args[1:] {
+		if a == "" {
+			a = "\"\""
```

Comment:
Is this necessary with the wrapper in Golang? I don't think that `exec.Command` removes empty parameters.